modules/@ergo/core/components/Form/JSONSchemaForm/JSONSchemaFormObject.vue

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <FormSection :title="title">
+        <Component
+            v-for="element in objectComponents"
+            :is="element.component"
+            v-bind="element.props"
+            :key="element.key"
+            @input="onValueChange" />
+    </FormSection>
+</template>
+
+<script>
+import { toCapitalize } from '@Core/models/stringWrapper';
+import FormSection from '@Core/components/Form/FormSection';
+
+export default {
+    name: 'JSONSchemaFormObject',
+    components: {
+        FormSection,
+    },
+    props: {
```

Comment:
use `inheritAttrs: false` if you use it in all JSONSchemaForm* components

Reply:
It's not good here. It will force re-rendering component each time we do change in form.